### DISCUSSION TITLE
is there a hotkey to trigger "send" in chat?

### DISCUSSION THREAD

**Discussion by siben168** [brianpetro/obsidian-smart-connections#938] [General]
I'd like to learn any shortcut key to trigger "send", quite annoying every time i should move my hand to the mouse. 

<img width="597" alt="image" src="https://github.com/user-attachments/assets/acb40c8b-bbf3-4417-a102-1c724bcde3f8" />

**Comment by brianpetro** [comment]
Tab + Enter is probably the most reliable. There are some more notes on this topic here https://github.com/brianpetro/obsidian-smart-connections/issues/163#issuecomment-2211470350 🎄

**Comment by siben168** [comment]
Thanks, Tab+Enter and CMD+Shift+Enter work for me. 👍
